src/dataspec/factories.py

```diff
@@ -179,7 +179,7 @@ def blankable_spec(
     Return a Spec which will validate values either by the input Spec or allow the
     empty string.
 
-    The returned Spec is equivalent to `s.any(spec, {""})`.
+    The returned Spec is equivalent to ``s.any(spec, {""})``.
```

Comment:
What is the meaning of 2 back ticks as opposed to 1?

Reply:
You'll see in my other PR #58 that I'm going all-in on reStructuredText (since that's what Sphinx uses). In rST, you use two backticks for inline code (which is the same as Markdown's single backtick).